export.py

```diff
@@ -1890,6 +1920,32 @@ def get_deformation(data_block, subframe, scene):
                 hairs = get_strands(scene, ob, psys)
                 data_block.motion_data.append((subframe, hairs))
 
+
+def is_transforming_recursive_brute_force(instance, instances):
+    if(instance is None):
+        return False
+    if(instance.transforming):
+        return True
+    elif instance.ob.parent is not None:
+        if(instances is None):
+            debug("debug","is_transforming_recursive_brute_force got None instances")
+            return False
+        return is_transforming_recursive_brute_force( instances[ instance.ob.parent.name ] )
+    else:
+        return False
+
+
+def vertices_equal(v1,v2):
+    return v1.co == v2.co
+# only compares vertices
+def meshes_are_equal(mesh1, mesh2):
+    if( len(mesh1.vertices) != len (mesh2.vertices) ):
+        return False
```

Comment:
In cache_motion when it's processing all cached deforms it checks (and did already) for different vertex counts and sets a mesh as non deforming if it's changing. It now tests edge and poly counts as well. It is not an expensive test, though I thought testing one was enough (when comparing meshes later it's still only comparing vertices).